eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -295,8 +305,8 @@ decodeCostModel lang = do
     Right cm -> pure cm
   where
     keys = case lang of
-      PlutusV1 -> PV1.costModelParamNames
-      PlutusV2 -> PV2.costModelParamNames
+      PlutusV1 -> Set.fromList $ Text.pack . showParamName <$> enumerate @PV1.ParamName
```

Comment:
This line gets repeated a few times in this commit, but I couldn't find a good place for it that doesn't feel completely arbitrary. I'm happy to extract it somewhere if anyone has suggestions on where it would best fit..

Reply:
I've addressed this comment